rare/components/tabs/games/env_vars.py

```diff
@@ -0,0 +1,98 @@
+from PyQt5.QtWidgets import QGroupBox, QTableWidgetItem
+from rare.ui.components.tabs.games.env_vars import Ui_env_vars_groupbox
```

Comment:
Use consistent naming scheme. Since you call the class `EnvVars` which is consistent with the naming of the file `env_var.py` the UI class should also be named `Ui_EnvVars`. This is controlled by the name of the base widget in Qt Designer.